libs/usb/HotplugAgent.cpp

```diff
@@ -77,25 +77,23 @@ AsyncronousLibUsbAdaptor *HotplugAgent::GetUSBAdaptor() const {
 }
 
 bool HotplugAgent::Init() {
-  if (libusb_init(&m_context)) {
-    OLA_WARN << "Failed to init libusb";
+  if (!LibUsbAdaptor::Initialize(&m_context)) {
     return false;
   }
 
-  OLA_DEBUG << "libusb debug level set to " << m_debug_level;
+  OLA_DEBUG << "libusb_set_debug(" << m_debug_level << ")";
   libusb_set_debug(m_context, m_debug_level);
 
   m_use_hotplug = HotplugSupported();
   OLA_DEBUG << "HotplugSupported(): " << m_use_hotplug;
-  if (m_use_hotplug) {
 #ifdef HAVE_LIBUSB_HOTPLUG_API
+  if (m_use_hotplug) {
     m_usb_thread.reset(new ola::usb::LibUsbHotplugThread(
           m_context, hotplug_callback, this));
-#else
-    OLA_FATAL << "Mismatch between m_use_hotplug and HAVE_LIBUSB_HOTPLUG_API";
```

Comment:
There shouldn't be any way for this message to occur so I removed it to make the code simpler.